Resources/config/checks/apc_fragmentation.xml

```diff
@@ -10,8 +10,8 @@
 
     <services>
         <service id="liip_monitor.check.apc_fragmentation" public="false" class="%liip_monitor.check.apc_fragmentation.class%">
-            <argument>%liip_monitor.check.apc_fragmentation.warning%</argument>
-            <argument>%liip_monitor.check.apc_fragmentation.critical%</argument>
+            <argument>__liip_monitor.check.apc_fragmentation.warning__</argument>
```

Comment:
I think using `%%` instead of `__` would be better. I saw it elsewhere (can't recall where).